### PR TITLE
Update masmas.js

### PR DESCRIPTION
**Pull request by N8python**
# Pull Request
Add isPrime and factors.

### CODE REVIEW

File: masmas.js [hunk 1]
```diff
@@ -139,6 +139,24 @@ function type(thing) {
 function validDate(str){
   return /\[(0[13578]\-[0-3][0-9]|0[469]\-[0-3][0-9]|02-[0-2][0-8]|(10|11|12)\-[0-3][0-9])\]/.test(str);
 }
+function isPrime(value) {
+    for(var i = 2; i < value; i++) {
+        if(value % i === 0) {
+            return false;
```
Comment (RDIL) [inline masmas.js]: Just do:
return value % i === 0

File: masmas.js [hunk 2]
```diff
@@ -139,6 +139,24 @@ function type(thing) {
...
+        }
+    }
+    return value > 1;
+}
+function factors(n){
```
Comment (RDIL) [inline masmas.js]: Add space

**Comment by RDIL** [comment]
@N8python please fix requested changes before merging

**Comment by N8python** [comment]
I didn't see what the changes did.

**Comment by RDIL** [comment]
@N8python my review above

**Pull request merged**